{{cookiecutter.project_name}}/dev-docker-compose.yml.default

```diff
@@ -0,0 +1,42 @@
+# Configuration for local development
+#
+# See README for setup and usage instructions
+#
+version: '3'
+
+services:
+  postgres:
+    image: postgres
+    ports:
+      - "5433:5433"
+      - "5432:5432"
+    volumes:
+      - "../db:/db"
+      - "../backup:/backup"
+      - ".:/project"
+    environment:
+      POSTGRES_PASSWORD: {{cookiecutter.project_name}}
+      POSTGRES_USER: {{cookiecutter.project_name}}
+      PGDATA: /db
+
+  redis:
+    image: redis
+    ports:
+      - "6379:6379"
+
+  django:
+    build:
+      context: ./
+      dockerfile: "./dev-docker/django/Dockerfile"
+    environment:
+      DJANGO_CONFIGURATION: Dev
+      DATABASE_HOST: postgres
+      DJANGO_SECRET_KEY: {{cookiecutter.secret_key}}
```

Comment:
Id rather not have the secret key written into the repo at all, this could be any string and we can generate a secret key for each deployed env and store that as a secret

Reply:
Agreed - secret key is now optional in settings for dev